pkg/diffs/diffs.go

```diff
@@ -166,7 +166,7 @@ func GetPresubmitsForCiopConfigs(prowConfig *prowconfig.Config, ciopConfigs conf
 				if env.ValueFrom.ConfigMapKeyRef == nil {
 					continue
 				}
-				if env.ValueFrom.ConfigMapKeyRef.Name == config.CiOperatorConfigsCMName {
+				if config.IsCiopConfigCM(env.ValueFrom.ConfigMapKeyRef.Name) {
```

Comment:
Wait so this method just returns _any_ presubmit that uses ci-op in a configmap mount? How's it used?

Reply:
The method returns any presubmit that references one of the configs in `ciopConfigs`. The rehearsal `main()` uses this method to obtain a set of jobs that were affected by a change to a ciop config - it passes just the ciop configs that were changed into this method.

Reply:
How is it doing the filtering for things actually in the `ciopConfigs`? Right now it returns any presubmit that mounts any CI Op config file?

Reply:
```
// if it is one of the CMs with ciop configs
if config.IsCiopConfigCM(env.ValueFrom.ConfigMapKeyRef.Name) {  
  // if the key into the CM is one of the filenames in `ciopConfigs`
  if _, ok := ciopConfigs[env.ValueFrom.ConfigMapKeyRef.Key]; ok {
    ret.Add(repo, job)
  }
}